src/components/dms/MessagesListHeader.tsx

```diff
@@ -157,9 +157,12 @@ function HeaderReady({
         moderation.ui('displayName'),
       )
 
-  const latestMessageFromOther = convoState.items.findLast(
-    item => item.type === 'message' && item.message.sender.did === profile.did,
-  )
+  const latestMessageFromOther = convoState.items
+    .reverse()
+    .find(
+      item =>
+        item.type === 'message' && item.message.sender.did === profile.did,
+    )
```

Comment:
Beware that `reverse()` will reverse the list in place. This could cause the list for be reversed on every render, causing the "found" item to be different every time.

If `findLast` doesn't work, you can clone the list before reversing it.

```suggestion
  const latestMessageFromOther = [...convoState.items]
    .reverse()
    .find(
      item =>
        item.type === 'message' && item.message.sender.did === profile.did,
    )
```